src/main/java/com/openshift/jenkins/plugins/pipeline/OpenShiftDeployer.java

```diff
@@ -43,17 +36,6 @@ public String getDepCfg() {
 		return depCfg;
 	}
 
-	public String getWaitTime() {
```

Comment:
same rebase / @oatmealraisin  PR concern - first blush let's defer wait time refactors to his PR unless you guys link and tell me you agree on how it will go